sql/src/test/java/org/apache/druid/sql/calcite/parser/DruidSqlParserUtilsTest.java

```diff
@@ -424,11 +425,11 @@ public void testConvertSqlNodeToGranularityWithIncorrectIngestionGranularityInTi
       args.add(new SqlIdentifier("__time", SqlParserPos.ZERO));
       args.add(SqlLiteral.createCharString("abc", SqlParserPos.ZERO));
       final SqlNode sqlNode = TimeFloorOperatorConversion.SQL_FUNCTION.createCall(args);
-      ParseException e = Assert.assertThrows(
-          ParseException.class,
+      DruidException e = Assert.assertThrows(
+          DruidException.class,
```

Comment:
It seems strange that `convertSqlNodeToGranularityThrowingParseExceptions` now also throws `DruidException` - maybe the function is not kept up-to-date and could use a rename to drop the exception suffix `convertSqlNodeToGranularity`.

Reply:
It does throw a parseException if the exception thrown by `convertSqlNodeToGranularity` is not of type DruidException. To avoid breaking things, would prefer leaving as is